aip/general/0135.md

```diff
@@ -232,6 +232,7 @@ exist, the service **must** error with `NOT_FOUND` (HTTP 404) unless
 
 ## Changelog
 
+- **2022-02-02**: Changed the etag error from FAILED_PRECONDITION (which becomes HTTP 400) to ABORTED (409).
```

Comment:
Okay, now I've looked into this a bit more, I'm happy with the change, but I think it would be good to indicate here that this change is to be consistent with then change to AIP-134 on 2021-03-05.

Reply:
have updated please see if this meets what you expected